src/py/mat3ra/made/tools/utils.py

```diff
@@ -322,3 +326,127 @@ def plane(plane_normal: List[float], plane_point_coordinate: List[float]):
             plane_normal=plane_normal,
             plane_point_coordinate=plane_point_coordinate,
         )
+
+
+class PerturbationFunctionHolder:
+    @staticmethod
+    def get_coord_transformation(perturbation_json: dict) -> Callable:
```

Comment:
Docstring
coord -> coordinate
This seems like a factory method, is it??

Reply:
yes, it is. Only had one option so far